src/locking/LockingManager.ts

```diff
@@ -82,6 +82,11 @@ export default class LockingManager {
     return true;
   }
 
+  public static async cleanUpLocks(): Promise<void> {
```

Comment:
To ensure that code will only be run on SCP, you can for example pass a boolean defaulting to true and guard the locks cleanup with it. Or guard the cleanups with `if (Configuration.isCloudFoundry() || Utils.isDevelopmentEnv())`. The latter have side effects but since it's on boolean variables, that's not an issue. I prefer the boolean as argument, so the code readers know the condition without have to read the method.  
Nitpick: cleanup is one word, so its use as camel case is more cleanupLocks.